tests/phpunit/CiviTest/CiviUnitTestCase.php

```diff
@@ -321,6 +321,8 @@ protected function setUp() {
     // disable any left-over test extensions
     CRM_Core_DAO::executeQuery('DELETE FROM civicrm_extension WHERE full_name LIKE "test.%"');
 
+    \CRM_Extension_System::singleton()->getManager()->enable(['org.civicrm.api4']);
+
```

Comment:
@eileenmcnaughton this is where this PR enables the extension. It should already be present and from what I can tell jenkins isn't failing to install it.